scripts/index.js

```diff
@@ -0,0 +1,109 @@
+
+
+const tempToColor = (temp) => { 
+    if (temp >= 80){
+        return "red-temp";
+    } else if (temp >= 70){
+        return "orange-temp";
+    } else if (temp >= 60){
+        return "yellow-temp";
+    } else if (temp >= 50){
+        return "green-temp";
+    } else {
+        return "teal-temp";
+    }
+    // ((temp>= 70) && (temp <= 79))  
+};
+
+const tempToLandscape = (temp) => {
+    if (temp >= 80){
+        return "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+    } else if (temp >= 70){
+        return "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"
+    } else if (temp >= 60){
+        return "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+    } else {
+        return "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    }
+};
+
+const skyToLandscape = (sky) => {
+    if (sky == "sunny"){
+        return "☁️ ☁️ ☁️ ☀️ ☁️ ☁"
+    } else if (sky == "cloudy"){
+        return "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+    } else if (sky == "rainy"){
+        return "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+    } else if (sky == "snowy"){
+        return "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+    }
+}
+
+const resetClickCount = ()=>{
+    StaticRange.clickCount = 0;
+    const townnameContainer = document.getElementBy("town-name");
+};
+
+const setTempClass = (tag, className) => {
```

Comment:
Clever! An alternative to removing all the classes we don't need, and adding the class we do need is using this attribute `tag.className = "red-temp"`